tests/ui/numbers-arithmetic/overflow-attribute-works-1.rs

```diff
@@ -0,0 +1,16 @@
+// check-pass
```

Comment:
```suggestion
// run-pass
```
Looks like you want the asserts to be actually checked here.